mllib/src/main/scala/org/apache/spark/mllib/clustering/PowerIterationClustering.scala

```diff
@@ -395,7 +395,7 @@ object PowerIterationClustering extends Logging {
     val points = v.mapValues(x => Vectors.dense(x)).cache()
     val model = new KMeans()
       .setK(k)
-      .setSeed(0L)
+      .setSeed(5L)
```

Comment:
Wait, but I don't get it, is this a test? This seems to be changing the main Power Iteration Clustering file. For the tests, we usually created data with very obvious clusterings where nearly any initialization would find the right thing. You should look over whichever test is failing without this change.